src/seedu/addressbook/data/person/Name.java

```diff
@@ -61,4 +61,9 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    @Override
+    public String getPrintableString() {
+	return getClass().getCanonicalName() + ": " + this.toString();
```

Comment:
this is good! but think which is more explicit and simpler to read using the word "Email" or `getClass().getCanonicalName()`